src/pages/overlay/components/overlay/ambassadors/ambassadors.module.scss

```
@@ -6,7 +6,7 @@
   left: 0;
   height: 100%;
   display: flex;
-
+  z-index: 0; // lets tooltips appear above the list
```

```suggestion
  z-index: 0; // lets tooltips appear above the list

```
nitpick, linebreak between blocks